oadr/oadr/manager/VENManager.cpp

```diff
@@ -83,11 +84,33 @@ IVENManager *VENManager::init(VENManagerConfig &config)
 				config.tls.sslVersion);
 	}
 
+	auto SignatureContextCreator = [&]()
```

Comment:
Nitpick: Use a method instead of lambda? Or even straight code

A lambda to call a constructor seems overkill

Reply:
I wanted to scope anything the logic of creating that SignatureContext, but agree that lamba may look strange in that context. Changed that to a free function in the unnamed namespace.